src/plugins/displayLayout/components/telemetry-view.scss

```diff
@@ -25,6 +25,10 @@
         margin-right: $interiorMargin;
     }
 
+    &.is-stale {
+        border: 1px solid cyan!important;
```

Comment:
Can we make this styling treatment a constant and include it in all the css files?

Reply:
I'll leave this in @rukmini-bose and @charlesh88 's court

Reply:
> Can we make this styling treatment a constant and include it in all the css files?

Absolutely, that's the plan.